Carbon/components/GetStartedButton.js

```diff
@@ -0,0 +1,33 @@
+import { View, Text, StyleSheet, Pressable } from 'react-native'
+import React from 'react'
+import { Colors } from '../colors/Colors'
+
+const GetStartedButton = ({onPress}) => {
+  return (
+    <View >
+        <Pressable style={styles.button} onPress={onPress} testID="getStartedButton">
+          <Text style={styles.buttonText}>Get Started</Text> 
+        </Pressable >
+   </View>
+
+  )
+}
+
+const styles = StyleSheet.create({
+  button: {
+    width: 200,
+    height: 40,
+    backgroundColor: Colors.primary.MINT,
+    borderRadius: 20,
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginTop: 10,
+    marginBottom: 30,
+  },
+  buttonText: {
+    color: 'white',
+    fontFamily: 'sans-serif',
```

Comment:
we've been having issues with sans-serif. We may need to come back and change the font.